python/lsst/summit/utils/astrometry/utils.py

```diff
@@ -259,10 +259,11 @@ def runCharactierizeImage(exp, snr, minPix):
     charConfig.doApCorr = False
     charConfig.doDeblend = False
     charConfig.repair.doCosmicRay = False
-    charConfig.repair.doInterpolate = True
+
     charConfig.detection.minPixels = minPix
     charConfig.detection.thresholdValue = snr
     charConfig.detection.includeThresholdMultiplier = 1
+    charConfig.detection.reEstimateBackground = False
```

Comment:
I'm curious about the reasoning for this.

Reply:
As discussed OOB, this was me not setting the detection background config correctly.

Reply:
And setting the detection background to the regular background config _does_ indeed work, just FYI.